ontotext-yasgui-web-component/src/services/yasgui/configuration/yasgui-configuration-builder.ts

```diff
@@ -109,12 +109,16 @@ export class YasguiConfigurationBuilder {
     config.yasguiConfig.yasr.yasrToolbarPlugins = yasrToolbarElements;
 
     // prepare the yasqe config
-    this.initShortcuts(config);
     config.yasguiConfig.yasqe.value = externalConfiguration.query || defaultYasqeConfig.query;
     config.yasguiConfig.yasqe.prefixes = NamespaceService.namespacesMapToArray(config.yasguiConfig.yasr.prefixes);
     config.yasqeConfig = {};
     config.yasqeConfig.initialQuery = externalConfiguration.initialQuery || defaultYasqeConfig.initialQuery;
     config.yasguiConfig.yasqe.createShareableLink = externalConfiguration.createShareableLink || defaultYasqeConfig.createShareableLink;
+    if (externalConfiguration.readonly !== undefined) {
+      config.yasguiConfig.yasqe.readOnly = externalConfiguration.readonly;
```

Comment:
Why the external and internla property name is different?

Reply:
fixed